azure-pipelines.yml

```diff
@@ -309,7 +309,7 @@ jobs:
 
       export PATH="$VERILATOR_PATH/bin:$PATH"
       fusesoc --cores-root=. \
-        run --target=sim --setup --build \
+        run --flag=fsel_top --target=sim --setup --build \
```

Comment:
I'd recommend changing the flags to `use_fileset_top`/ `use_fileset_ip` (or just `fileset_top` / `fileset_ip`) instead. It also matches with fusesoc keyword `filesets`.